server/src/main/java/org/opensearch/common/util/BytesRefHash.java

```diff
@@ -6,179 +6,291 @@
  * compatible open source license.
  */
 
-/*
```

Comment:
Please bring the header back, we are modifying the original file, it is fair to preserve the licensing

Reply:
Taken care of, yet to push my local changes. This got unknowingly removed when I replaced the original implementation with the new `CompactBytesRefHash`.